app/cdap/components/InlayDrawerWidget/InlayDrawerWidget.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { action } from '@storybook/addon-actions';
+import InlayDrawerWidget, { IMenuItem } from 'components/InlayDrawerWidget';
+import React from 'react';
+
+export default {
+  title: 'InlayDrawerWidget',
+  component: InlayDrawerWidget,
+};
+
+const open = true;
+
+const handleDrawerCloseIconClick = () => action('clicked')('Drawer Closed');
+const onSaveButtonClick = () => action('clicked')('Save Button Clicked');
+const onApplyButtonClick = () => action('clicked')('Apply Button Clicked');
+const onDownloadButtonClick = () => action('clicked')('Download Button Clicked');
+
+const actionsOptions: IMenuItem[] = [
+  {
+    label: 'Save',
+    value: 'save',
+    clickHandler: onSaveButtonClick,
+  },
+  {
+    label: 'Apply',
+    value: 'apply',
+    clickHandler: onApplyButtonClick,
+  },
+  {
+    label: 'Download',
+    value: 'download',
+    clickHandler: onDownloadButtonClick,
+  },
+];
+
+export function Default(args) {
```

Comment:
Why are the exports so weird here?

Reply:
@seanfeld 

I am exporting the component here, the component name is Default as this will be the Default version of the Inlay Drawer Widget

so, `export function Default(args) {`

args is the props that the component will accept
and in order for the knobs and actions to render in the storybook,
we have to use the `ComponentName.args = { ... }`

here, `Default.args = { ... }`

I can change the name of the component to DefaultInlayDrawerWidget if this doesn't seem right
Please confirm